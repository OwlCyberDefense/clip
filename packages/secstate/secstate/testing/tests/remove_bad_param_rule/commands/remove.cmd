secstate remove PassComp-R-2-2
