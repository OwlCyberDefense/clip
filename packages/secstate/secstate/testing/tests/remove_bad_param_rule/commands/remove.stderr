Could not find PassComp-R-2-2 in database
